files/en-us/learn/accessibility/what_is_accessibility/index.html

```diff
@@ -78,13 +78,13 @@ <h3 id="People_with_visual_impairments">People with visual impairments</h3>
 
 <p>In terms of statistics, the World Health Organization estimates that "285 million people are estimated to be visually impaired worldwide: 39 million are blind and 246 million have low vision." (see <a href="https://www.who.int/mediacentre/factsheets/fs282/en/">Visual impairment and blindness</a>). That's a large and significant population of users to just miss out on because your site isn't coded properly — almost the same size as the population of the United States of America.</p>
 
-<h3 id="People_with_hearing_impairments">People with hearing impairments</h3>
+<h3 id="People_with_hearing_impairments">Deaf people</h3>
 
-<p>Otherwise known as people with auditory impairments or deaf people, this group of people has either low hearing levels or no hearing at all. Hearing-impaired people do use ATs (see <a href="https://www.nidcd.nih.gov/health/assistive-devices-people-hearing-voice-speech-or-language-disorders">Assistive Devices for People with Hearing, Voice, Speech, or Language Disorders</a>), but there are not really special ATs specific for computer/web use.</p>
+<p><a href="https://www.nad.org/resources/american-sign-language/community-and-culture-frequently-asked-questions/">Deaf and hard-of-hearing (DHH)</a> people have various levels of hearing loss ranging from mild to profound. Although some do use AT (see <a href="https://www.nidcd.nih.gov/health/assistive-devices-people-hearing-voice-speech-or-language-disorders">Assistive Devices for People with Hearing, Voice, Speech, or Language Disorders</a>), they are not widespread. Recently, <a href="https://blog.google/products/chrome/live-caption-chrome/">Google Chrome released an auto-captioning extension</a></p>.
```

Comment:
```suggestion
<p><a href="https://www.nad.org/resources/american-sign-language/community-and-culture-frequently-asked-questions/">Deaf and hard-of-hearing (DHH)</a> people have various levels of hearing loss ranging from mild to profound. Although some do use AT (see <a href="https://www.nidcd.nih.gov/health/assistive-devices-people-hearing-voice-speech-or-language-disorders">Assistive Devices for People with Hearing, Voice, Speech, or Language Disorders</a>), they are not widespread.</p>
```

Yes, Google Chrome released an auto-captioning extension. From what I hear, the quality is sub-par and not representing the content on an acceptable level (compared to a manual translation). That might be different for US content, where you have a broader corpus to Machine Language-translate from.

I'm cautious to advertise it. Perhaps add it in a „See also” section?